packages/web/src/pages/Home.tsx

```
@@ -96,6 +98,18 @@ export default function Home() {
   const hasMessages = events.length > 0;
 
+  useEffect(() => {
+    const state = location.state as { resumeChat?: boolean } | null;
+    if (state?.resumeChat) {
+      return;
+    }
+    globalChat.startNewSession();
+    // Run only when Home is entered. `globalChat` is intentionally omitted
+    // because it changes as events stream in; including it would clear the
+    // conversation on every chat update.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)